scripts/generate-releases.sh

```diff
@@ -146,14 +137,16 @@ generate_release_resources() {
             "metadata": {
               "annotations": .metadata.annotations + {"acs.redhat.com/original-snapshot-name": "'"${snapshot}"'"},
               "labels": .metadata.labels,
-              "name": "'"${snapshot_copy}"'",
+              "name": "'"${snapshot_copy_name}"'",
               "namespace": .metadata.namespace
             },
             "spec": .spec
            }'
 
         application="$(echo "$line" | cut -d "|" -f 2)"
-        release_name_with_application="${application}-${release_name_suffix}"
+
+        # Crop the release name to 51 characters to avoid the Kubernetes limit of 63 characters for re-runs.
```

Comment:
Suggestion:
```suggestion
        # Crop the release name to 51 characters to avoid exceeding the Kubernetes limit of 63 characters for re-runs.
```
or something like that.